### PR TITLE
BOM-2798: Upgrade Credentials to Django 3.2

### PR DESCRIPTION
**Pull request by mzulqarnain1**
## Description

- Upgraded To Use Django 3.2
- Dropped Django 2.2 Support
- Ran Make Upgrade

## Ticket Link

Link to the associated ticket
[BOM-2798](https://openedx.atlassian.net/browse/BOM-2798)

### CODE REVIEW

**Comment by awais786** [comment]
I am not sure but is there any chance of templates coming from DB ? If yes need to check the `{% load staticfiles %}` tag in those entries.

**Review by awais786 — APPROVED** [review]

**Comment by staubina** [comment]
> I am not sure, is there any chance of templates coming from DB ? If yes need to check the {% load staticfiles %} tag in those entries.

@awais786 We will look into the DB templates, I think those are only for Course Certs in the LMS though. 

Did you look at https://github.com/edx/credentials-themes though?

File: requirements/constraints.txt [hunk 1]
```diff
@@ -11,5 +11,7 @@
 # Common constraints for edx repos
 -c common_constraints.txt
 
+Django<4.0
```
Comment (justinhynes) [inline requirements/constraints.txt:14]: Should this be `Django<3.3` to stay on the LTS version?

**Review by justinhynes — DISMISSED** [review]
At first glance, this looks good and seems to work fine locally. 

I have a few questions to make sure we are constraining the versions of Django to the LTS versions.

**Comment by mzulqarnain1** [comment]
> > I am not sure, is there any chance of templates coming from DB ? If yes need to check the {% load staticfiles %} tag in those entries.
> 
> @awais786 We will look into the DB templates, I think those are only for Course Certs in the LMS though.
> 
> Did you look at https://github.com/edx/credentials-themes though?

Yes @staubina We have upgraded the templates in [credentials-themes ](https://github.com/edx/credentials-themes ) and the tag is bumped in this PR.

File: requirements/constraints.txt (hunk 1 above)
Comment (justinhynes) [reply requirements/constraints.txt:14]: I think my concern is all set for now according to this: https://www.djangoproject.com/download/. It doesn't look like there are any versions planned past 3.2.x, and they will move to Django 4.x next. According to Django's plans, I think this constraint is fine.

**Review by Tj-Tracy — APPROVED** [review]

**Pull request merged**